README.md

```diff
@@ -90,7 +90,7 @@ Git actually destroys the old commits and replaces them with new commits
 
 This is one of the things that can make `git rebase` dangerous,
  and it's the reason why you never rebase code that's already been
- published and shared - you run the risk of breaking other peoples' code.
+ published and shared - you run the risk of breaking other peoples' code. After a rebase, pushing to the branch will require --force-with-lease since the history has been rewritten.
```

Comment:
Add Antony's language " rebase adds commits onto a shared commit in both branches history and then reapplies your commits on top of those." 
I don't think they should be taught how to force push, I think it will lead to plroblems.